.github/workflows/llm_performance_tests.yml

```diff
@@ -108,6 +108,7 @@ jobs:
           python -m pip install --upgrade einops
           python -m pip install --upgrade transformers_stream_generator
           python -m pip install --upgrade tiktoken
+          python -m pip install --upgrade transformers
```

Comment:
specify version ==4.34.0? maybe in a next PR. also add a comment 4.34.0 is for mistral

Reply:
ok